tests/update_reference_data.sh

```diff
@@ -126,6 +126,7 @@ tests[polymer_injectivity]="flow polymer_injectivity 2D_POLYMER_INJECTIVITY"
 tests[nnc]="flow editnnc NNC_AND_EDITNNC nnc"
 tests[udq_wconprod]="flow udq_actionx UDQ_WCONPROD udq_wconprod"
 tests[udq_actionx]="flow udq_actionx UDQ_ACTIONX udq_actionx"
+tests[udq_M1]="flow udq_actionx UDQ_M1 udq_M1"
```

Comment:
"flow udq_actionx UDQ_M1 udq_uadd"

Reply:
OK thank you; I am trying that now. But I do not understand what was wrong with my initial attempt.

As time has gone by the common pattern is that the directories have multiple test files in them, maybe the way to add testdata should treat/describe that as the normal (only ??) way to do it?

Reply:
the last parameter must match the cmake test name